.github/workflows/ci.yml

```diff
@@ -0,0 +1,90 @@
+---
```

Comment:
Rather than syncing in the large CI workflows only to remove them in the next, should we update this to current modulesync_config master? Or if you want, we can make it a new release of msync_config.

Reply:
did a new modulesync against master and rebased #164 . tests passed so I'm going to release modulesync_config 5.2.0